sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java

```diff
@@ -61,19 +64,50 @@
  * @since 5.3
  */
 public interface WsClient {
-  OrganizationService organizations();
+
+  /**
+   * @deprecated since 7.0 use {@link #organizations()} instead
+   */
+  @Deprecated
+  OrganizationService organizationsOld();
+
+  OrganizationsService organizations();
+
+  /**
+   * @deprecated since 7.0 use {@link #components()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.component.ComponentsService componentsOld();
 
   ComponentsService components();
 
   FavoritesService favorites();
 
+  /**
+   * @deprecated since 7.0 use {@link #issues()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.issue.IssuesService issuesOld();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)